packages/components/src/guide/style.scss

```diff
@@ -92,12 +87,15 @@
 	height: 80vh;
 	max-height: 575px;
 
-	@media (max-width: $break-small) {
-		bottom: 5%;
-		left: $grid-unit-20;
-		right: $grid-unit-20;
-		top: 5%;
-		margin: 0 auto;
+	@media ( max-width: $break-small ) {
+		margin: auto;
+		min-width: 280px;
+		max-width: calc(100vw - #{$grid-unit-20} * 2);
+
+		// This can be avoided if we add styles to let the image resize or don't change the min-width
+		&.components-modal__content {
+			overflow-x: hidden;
+		}
```

Comment:
My previous suggestion was botched. The `&` needed a space after it or to not be there at all. Besides, if we were to use this it would go better I think in the ruleset at line 9.
```suggestion
```